pkg/operator/starter.go

```diff
@@ -23,12 +26,15 @@ const (
 	operatorName     = "openstack-cinder-csi-driver-operator"
 	operandName      = "openstack-cinder-csi-driver"
 	instanceName     = "cinder.csi.openstack.org"
+
+	CloudConfigNamespace = "openshift-config"
+	CloudConfigName      = "cloud-provider-config"
```

Comment:
I just noticed that we already have this config map in "openshift-cluster-csi-drivers" namespace created by CSO: https://github.com/openshift/cluster-storage-operator/blob/c76dd10bd5370239e320500da68782d6e8d3326a/pkg/operator/csidriveroperator/csioperatorclient/manila.go#L64-L73
So, there is no reason to create it again

Reply:
Sweet. I removed the configmap sync'ing from the patch, that leaves us with only mounting the certificate to the needed pods.